source/adios2/toolkit/transport/wan/WANZmq.cpp

```diff
@@ -22,7 +23,7 @@ WANZmq::WANZmq(const std::string ipAddress, const std::string port,
 : Transport("wan", "zmq", mpiComm, debugMode), m_IPAddress(ipAddress),
   m_Port(port)
 {
-
+    m_Context = zmq_ctx_new();
```

Comment:
Does this need to be checked for validity in case it fails or will it always be guaranteed to succeed?